plugins/source-kafka/pkg/client/client.go

```diff
@@ -28,22 +28,19 @@ import (
 )
 
 type kafkaSourceClient struct {
-	kafkaSourceParams *types.KafkaSourceParams
-	client            clientv1alpha1.SourcesV1alpha1Interface
 	namespace         string
+	kafkaSourceParams *types.KafkaSourceParams
 	knSourceClient    sourcetypes.KnSourceClient
```

Comment:
IMO  we should rename this to  `PluginSourceClient` to better reflect the scope and clear separation with core kn clients. (could be done in a separate PR). WDYT ?

Reply:
Makes sense. But this comes from the common code. Let’s do these change when I move the `kn-source-pkg` code. Cool?